### DISCUSSION TITLE
alarme s'affiche au bout de deux minutes

### DISCUSSION THREAD

**Discussion by pinea81** [arkypita/LaserGRBL#1072] [Q&A]
LaserGrbl v3.5.3.0
Grbl v1.1f
Wrapper: UsbSerial
Win 7, Home Premium, 6.1.7601.65536, Service Pack 1 (7)CLR: C:\Windows\Microsoft.NET\Framework64\v2.0.50727\mscorlib.dll 2.0.50727.8806 (QFE.050727-8800)

TypeOf exception  [System.ObjectDisposedException]
Exception message [Le handle sécurisé a été fermé]
Exception source  [mscorlib], thread []
Exception method  [IntPtr SafeHandleC2NHelper(System.Object, IntPtr)]

   ----------- stack trace -----------
   à System.StubHelpers.StubHelpers.SafeHandleC2NHelper(Object pThis, IntPtr pCleanupWorkList)
   à Microsoft.Win32.UnsafeNativeMethods.GetOverlappedResult(SafeFileHandle hFile, NativeOverlapped* lpOverlapped, Int32& lpNumberOfBytesTransferred, Boolean bWait)
   à System.IO.Ports.SerialStream.EventLoopRunner.WaitForCommEvent()
   à System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   à System.Threading.ThreadHelper.ThreadStart()

**Comment by arkypita** [comment]
Thanks for this information.
Unfortunately, this problem is driving me crazy. This is a problem in the microsoft code that occurs in very rare conditions and that I can't simulate, so I struggle to solve.

It has already been reported by other users, but almost nobody has been able to describe the situation / condition with which it occurs.

Is it a repetitive problem? Is there any procedure that makes it happen?

**Comment by pinea81** [comment]
bonjour, j'ai préférer passer sur un ordinateur qui fonctionne avec Windows 10 , mais maintenant le cycle se coupe toujours au bout d'une minute et le message d'erreur est : unexpected reset
merci

**Comment by arkypita** [comment]
Hi
I write a new SerialUSB wrapper to comunicate with the engraver, that replace microsoft implementation that has some bug like the one reported here. 

If you want you can test by download from here: https://github.com/arkypita/LaserGRBL/releases/download/v3.8.10/install.exe and enable UsbSerial2 connection protocol (before connect!).

![image](https://user-images.githubusercontent.com/8782035/103441483-9ff92f00-4c4e-11eb-963b-18b2de8a466b.png)
